### ISSUE TITLE
Difference for embedded lens-correction for OM-System Mark II between DNG and ORF

### ISSUE THREAD

**Issue by astmuc** [darktable-org/darktable#16648]
### Describe the bug

Camera: OM-System Mark II

I can see the automatic lens correction using embedded info if the input is DNG. But i see no lens correction for the native ORF format.

Is this an issue or simply an unsupported feature ?

### Steps to reproduce

* load OM-System Mark II ORF file and check for automatic lens correction -> no
* Convert this file with Adobe DNG Converter and load it as DNG -> automatic lens correction is shown

### Expected behavior

darktable should apply embedded lens correction of an ORF file

### Where did you obtain darktable from?

distro packaging

### darktable version

darktable 4.7.0~git976.28ab0294-1+11580.1

### What OS are you using?

Linux

### What is the version of your OS?

Ubuntu 22.04

### Describe your system?

AMD Ryzen 9 7900 12-Core, 128 Gb, Nvidia 4070 super

### Are you using OpenCL GPU in darktable?

Yes

### If yes, what is the GPU card and driver?

nvidia 4070 Super, driver version 550

**Comment by kmilos** [comment]
See https://github.com/darktable-org/darktable/pull/12760

**Comment by kmilos** [comment]
Ah, it is also possible your snap package uses an outdated version of exiv2 that doesn't support OM System rebranding from Olympus... Are you also missing any other metadata perhaps? What do you think @paolodepetrillo ?

FYI, if that is the case, for any problems w/ the snap package of  such kind you will need to contact whoever made it available, the darktable project is not responsible for it.

**Comment by astmuc** [comment]
I do not understand yet your argument. I have no snap installed. It is the daily build installed from the ubuntu installation package offered in darktable.org. See this screenshot
<img width="247" alt="Screenshot_20240420_132925" src="https://github.com/darktable-org/darktable/assets/37765839/f3fa86b7-029d-46fd-8e36-7f8558f007e0">
Do you mean this with "snap" ?

**Comment by kmilos** [comment]
Sorry, mixed up w/ another recent issue where snap was apparent...

Anyhow, I think the default exiv2 version has n 22.04 does not support OM System cameras well, you need at least 0.26.6 IIRC...

**Comment by astmuc** [comment]
As you are tkaing so much time for my problem, i dare to ask something here. I managed to run darktable with a local build from master inluding the the latest exiv2 lib. I installed it into /usr/local/lib and cmake found it there. This darktable shows me the lens correction.

If i start darktable like this
```
LD_DEBUG=libs darktable 2> debug.log
```
to get the LD library-search  resolution log i see that darktable explicitly looks for
```
    133301:     find library=libexiv2.so.27 [0]; searching
    133301:      search path=/usr/local/lib             (LD_LIBRARY_PATH)
    133301:       trying file=/usr/local/lib/libexiv2.so.27
    133301:      search cache=/etc/ld.so.cache
    133301:       trying file=/lib/x86_64-linux-gnu/libexiv2.so.27
```
I read this as if darktable wants to have a 2.7.x version and therefore does not include my installation of libexiv2.so. Is this possible ? Or is it an installation problem of my original exiv2 installation ? I did this with ubuntu apt.
